packages/store-sync/src/recs/defineInternalComponents.ts

```diff
@@ -4,10 +4,10 @@ import { Table } from "../common";
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
 export function defineInternalComponents(world: World) {
   return {
-    TableMetadata: defineComponent<{ table: Type.T }, Metadata, Table>(
+    Tables: defineComponent<{ table: Type.T }, Metadata, Table>(
       world,
       { table: Type.T },
-      { metadata: { componentName: "TableMetadata" } }
+      { metadata: { componentName: "Tables" } }
```

Comment:
I think this got renamed by mistake!

Reply:
Might be worth adjusting this name anyway to like `RegisteredClientTables` or something

Reply:
That said, I wonder if the now-built-in `Tables` component (derived from mud config) would have everything I need (minus some nice-to-haves like an already parsed table schema).

Reply:
good catch!